### ISSUE TITLE
[Tokens] Holders and Token Transfers fields are missing from the header (top panel).

### ISSUE THREAD

**Issue by 1Mateus** [hack-a-chain-software/indexer-kadena#61]
## EN 
Already have the transfers, but not the holders.
- [ ]  The frontend should be updated to query the token module and display the balances of accounts that currently hold the token.
## PT
 já tem transfers, mas não tem holders. 
- [ ] Tem que implementar no front, fazendo “where” pelo modulo do token, mostrando o balance das contas que holdam o token. 

https://www.notion.so/Mat-Reply-Hackchain-Indexer-Explorer-Frontend-review-fa7e71e82bdf44cd9fc99821b484fb88?pvs=4#122254f646f643d1a350a5919d3474c8

**Comment by 1Mateus** [comment]
@davibauer  Você consegue me dar um help para entender como funcionaria essa query? isso vai valeria para todas as chains? pq salvamos cada balance separado, certo? teria como agrupar.

Aqui o design da tela:

![Image](https://github.com/user-attachments/assets/e9982fc3-178c-42d5-a3b7-4a2c9b00e583)

**Comment by davibauer** [comment]
@1Mateus Sim, teria que agrupar, melhor disponibilizar uma query separada, que faz os joins nas tabelas e tu apenas consumir o método pronto no endpoint. Montar algo assim no GraphQL eu imagino que seja inviável por conta da performance e da complexidade.

**Comment by davibauer** [comment]
@1Mateus Acabei de criar a query `getHolders`. Exemplo de uso:

Request
`query getHolders {
  getHolders(first: 5, moduleName: "coin") {
    edges {
      cursor
      node {
        address
        quantity
        percentage
      }
    }
    pageInfo {
      endCursor
      hasNextPage
      hasPreviousPage
      startCursor
    }
  }

}
`

Response
`{
  "data": {
    "getHolders": {
      "edges": [
        {
          "cursor": "MQ==",
          "node": {
            "address": "81387a6c0f3f362fa578a11f65624d78f77d326d93d943de72994dece6cae7d4",
            "quantity": 4845850,
            "percentage": 35.59830586194276
          }
        },
        {
          "cursor": "Mg==",
          "node": {
            "address": "k:0d584a4cbbfd9a4878d816512894e65918e54fae13df39a6f520fc90caea2fb0",
            "quantity": 2111247,
            "percentage": 15.509521849852776
          }
        },
        {
          "cursor": "Mw==",
          "node": {
            "address": "",
            "quantity": 1453274,
            "percentage": 10.675958263918407
          }
        },
        {
          "cursor": "NA==",
          "node": {
            "address": "k:bfcfdd740857ab6be8d6acce80f9fef6c0ad99e46c9b6e2dc24e0fe2850bede9",
            "quantity": 1421366,
            "percentage": 10.44155754094042
          }
        },
        {
          "cursor": "NQ==",
          "node": {
            "address": "k:531cbccba6924bcc83063e9f7880698c77e96e1b10c6ff4d3d57e199aa436760",
            "quantity": 360020,
            "percentage": 2.6447583141072535
          }
        }
      ],
      "pageInfo": {
        "endCursor": "NQ==",
        "hasNextPage": true,
        "hasPreviousPage": false,
        "startCursor": "MQ=="
      }
    }
  }
}`

**Comment by rafaeldkb** [comment]
Aparentemente, tá show!
